### PR TITLE
Project Deployed

### PR DESCRIPTION
**Pull request by JAAR91**
- Project has been deployed and it's working on Heroku
- Master Branch has been updated with development
- App is being deployed from master branch
- https://secret-plains-32456.herokuapp.com/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
 
 ### Deployment
 
-Wesite:[Heroku](https://secret-plains-32456.herokuapp.com/)
+Wesite: [Heroku](https://secret-plains-32456.herokuapp.com/)
 
 
 ## Getting Started
```
Comment (emmanuellekamwa) [inline README.md:15]: The getting started section of your README is not explicit enough. You have to describe how someone can have your project cloned on his/her local machine.

Comment (emmanuellekamwa) [inline README.md:22]: The getting started section of your readme file is not explicit enough. You have to describe how someone can get your project on his/her local machine. This is for the sake of people who are new to github and may not know what to do so as to have a look on your project.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
Hi team. You have done an impressive work so far 👏🏽👏🏽👨🏽‍💻👨🏽‍💻👌🏽👌🏽 and I really like the other features that you have added to the project. The only thing to adjust to get this milestone approved is just to add some more information on the getting started section of your readme file. When you are done just submit it again for another code review. Good luck guys 😉✌👨🏽‍💻

**Review by GzimAsani — APPROVED** [review]
# STATUS: MIllstone Accepted :green_circle: 

## Hello @JAAR91 thank you for making all the required changes, you project now looks complete and good luck with the next projects :rocket: 

# HAPPY CODING :computer: :desktop_computer:

**Pull request merged**